superset/assets/visualizations/EventFlow.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+  App,
+  withParentSize,
+  cleanEvents,
+  TS,
+  EVENT_NAME,
+  ENTITY_ID,
+} from '@data-ui/event-flow';
+
+/*
+ * This function takes the slice object and json payload as input and renders a
+ * responsive <EventFlow /> component using the json data.
+ */
+function renderEventFlow(slice, json) {
+  const container = document.querySelector(slice.selector);
+  const hasData = json.data && json.data.length > 0;
+
+  // the slice container overflows ~80px in explorer, so we have to correct for this
+  const isExplorer = (/explore/).test(window.location.pathname);
```

Comment:
I tried this to begin with but it doesn't seem to work/couldn't come up with any fix for the css. The panel dom looks like:
```html
<div class="panel">
  <div class="panel-heading">...</div>
  <div class="panel-body">...</div>
</div>
```
`panel-body` is what the slice is in and it seems to have the same height as the parent `panel`. that means it overflows by the height of `panel-heading` :/